com.logseq.Logseq.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
I haven't because they don't have a way to install a package with it yet. They currently only distribute AppImage and zip. The Exec line will be tricky.

Reply:
the exec line isn't a problem it can be updated at install time.

Reply:
I mean the upstream doesn't use it. Maybe it can be put under some `contrib` directory and some readme to explain how to install. I'll open an issue with them.